src/modules/ekf2/ekf2_main.cpp

```diff
@@ -417,10 +417,10 @@ Ekf2::Ekf2():
 	_mag_bias_id(this, "MAGBIAS_ID"),
 	_mag_bias_saved_variance(this, "MAGB_VREF"),
 	_mag_bias_alpha(this, "MAGB_K"),
-	_acc_bias_lim(this, "EKF2_ABL_LIM", false, _params->acc_bias_lim),
-	_acc_bias_learn_acc_lim(this, "EKF2_ABL_ACCLIM", false, _params->acc_bias_learn_acc_lim),
-	_acc_bias_learn_gyr_lim(this, "EKF2_ABL_GYRLIM", false, _params->acc_bias_learn_gyr_lim),
-	_acc_bias_learn_tc(this, "EKF2_ABL_TAU", false, _params->acc_bias_learn_tc),
+	_acc_bias_lim(this, "ABL_LIM", true, _params->acc_bias_lim),
```

Comment:
Why are you changing the names of these parameters?

Reply:
It's using the parent prefix (third param = true). It saves a few characters but I'm really not a fan of it, because:
- you cannot grep the source to find where a param is used. This already caught me by surprise.
- it adds confusion and makes the API more complicated than needed.

Reply:
I'm fine with it either way as long as it's consistent per module (this wasn't).